stable/kibana/templates/configmap.yaml

```diff
@@ -1,7 +1,8 @@
+{{- if ne .Values.configMap.override true -}}
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ template "kibana.fullname" . }}
+  name: {{ template "kibana.name" . }}
```

Comment:
Why the switch from `fullname` to `name` here if this configmap is only triggered when the override is false? In that scenario this would be a breaking change for users.